src/main/java/com/ita/if103java/ims/dao/impl/DashboardDaoImpl.java

```diff
@@ -83,10 +87,20 @@ public DashboardDaoImpl(WarehouseLoadRowMapper warehouseLoadRowMapper,
     }
 
     @Override
-    public List<EndingItemsDto> findEndedItemsByAccountId(int minQuantity, Long accountId) {
+    public Page<EndingItemsDto> findEndedItemsByAccountId(Pageable pageable, int minQuantity, Long accountId) {
         try {
-            return jdbcTemplate.query(Queries.SQL_FIND_ENDED_ITEMS_BY_ACCOUNT_ID, endingItemsRowMapper,
-                minQuantity, accountId);
+            String sort = pageable.getSort().stream().map(
+                x -> x.getProperty() + " " + x.getDirection().name()).collect(Collectors.joining(", "));
```

Comment:
Please wait, when
https://github.com/IF-103-Java/ims/pull/169/files
will be merged and than use the same method from the JDBCUtil